spec/api-desktop-capturer-spec.js

```diff
@@ -22,91 +24,62 @@ describe('desktopCapturer', () => {
     }
   })
 
-  it('should return a non-empty array of sources', done => {
```

Comment:
how do we make sure we don't break the legacy callback based APIs if the tests are removed? maybe keep the old ones and remove them in the next major version with the API cleanup.

Reply:
done